staging/src/k8s.io/api/flowcontrol/v1beta3/types.go

```diff
@@ -103,10 +103,25 @@ const (
 	AutoUpdateAnnotationKey = "apf.kubernetes.io/autoupdate-spec"
 )
 
+const (
+	// This annotation applies to v1beta3 only.
+	//
+	// The presence of this annotation in a v1beta3 object means that
+	// a zero value in the 'NominalConcurrencyShares' field means zero
+	// rather than the old default of 30.
+	//
+	// To set a zero value for the 'NominalConcurrencyShares' field in v1beta3,
+	// set the annotation to an empty string:
+	//   "flowcontrol.k8s.io/concurrency-shares-defaults-to-zero": ""
+	//
+	PriorityLevelConcurrencyShareDefaultKey = "flowcontrol.k8s.io/concurrency-shares-defaults-to-zero"
```

Comment:
This annotation is specific to v1beta3... we no longer serve or store in v1beta1 or v1beta2, so we don't have to worry about round-tripping zero values to those, and we don't want this annotation to be used in v1 (see comment about forbidding it in validation after we've converted to the internal version). I'd suggest putting v1beta3 in the annotation key and documenting it is only for use in v1beta3.

Also, "defaults-to-zero" isn't quite right, when this annotation is present we *don't* default zero.

I'd suggest `flowcontrol.k8s.io/v1beta3-preserve-zero-concurrency-shares`